etc/schema.json

```diff
@@ -539,10 +539,20 @@
                                 "type": "integer"
                             },
                             "uuid": {
-                                "type": "string"
+                                "type": "string",
+                                "suricata": {
+                                    "keywords": [
+                                        "dcerpc.iface"
```

Comment:
Wondering if this belongs to root object `interfaces` ...

Reply:
There is no `ack_result` in the keyword, that's why, I added it to the other two children only. lmk wdyt?

Reply:
No big deal, this shows the complexity of the pairing task.

Why is `ack_result` in interfaces ? I do not feel the reason for it...

Reply:
> Why is ack_result in interfaces ? I do not feel the reason for it...

I don't really know. I couldn't find what it can be used for easily but I do see it can hold different values in some dcerpc pcaps. I think I added this because it exists is smb/dcerpc as well..

Reply:
I agree this field exists on the wire, but I am not sure it belongs in the `interfaces` object... But let's not spend too much time on this